doc/source/reference/udf/mroonga_highlight_html.rst

```diff
@@ -17,12 +17,14 @@ Syntax
 
 ``mroonga_highlight_html()`` has required parameter and optional parameter::
 
-  mroonga_highlight_html(text, query AS query)
+  mroonga_highlight_html(text, query AS query [, open_tag AS open_tag, close_tag AS close_tag])
 
-  mroonga_highlight_html(text, keyword1, ..., keywordN)
+  mroonga_highlight_html(text, keyword1, ..., keywordN [, open_tag AS open_tag, close_tag AS close_tag])
```

Comment:
How about listing multiple syntaxes instead of mixing syntaxes?

```rst
  mroonga_highlight_html(text, query AS query)
  mroonga_highlight_html(text, query AS query, open_tag AS open_tag)
  mroonga_highlight_html(text, query AS query, close_tag AS close_tag)
  mroonga_highlight_html(text, query AS query, open_tag AS open_tag, close_tag AS close_tag)
  
  mroonga_highlight_html(text, keyword1, ..., keywordN)
  mroonga_highlight_html(text, keyword1, ..., keywordN, open_tag AS open_tag)
  mroonga_highlight_html(text, keyword1, ..., keywordN, open_tag AS close_tag)
  mroonga_highlight_html(text, keyword1, ..., keywordN, open_tag AS open_tag, close_tag AS close_tag)
```

Or spliting named arguments:

```rst
  mroonga_highlight_html(text, query AS query)
  mroonga_highlight_html(text, query AS query, named_option1, ..., named_optionN)
  
  mroonga_highlight_html(text, keyword1, ..., keywordN)
  mroonga_highlight_html(text, keyword1, ..., keywordN, named_option1, ..., named_optionN)

Named options:

  * `AS open_tag`
  * `AS close_tag`
```

Reply:
I have modified it by the former case.